examples/massive-workflow.yaml

```diff
@@ -0,0 +1,47 @@
+# massive workflows
+# 500 DAGs with 9 tasks a piece
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  name: massive-workflow
+spec:
+  entrypoint: main
+  templates:
+    - name: main
+      steps:
+        - - name: dag
+            template: dag
+            withSequence:
+              count: "500"
+    - name: dag
+      dag:
+        tasks:
+          - name: A
+            template: whalesay
+          - name: B
+            template: whalesay
+            dependencies: [A]
+          - name: C
+            template: whalesay
+            dependencies: [A]
+          - name: D
+            template: whalesay
+            dependencies: [B, C]
+          - name: E
+            template: whalesay
+            dependencies: [B, C]
+          - name: F
+            template: whalesay
+            dependencies: [B, C]
+          - name: G
+            template: whalesay
+            dependencies: [D, E, F]
+          - name: H
+            template: whalesay
+            dependencies: [D, E, F]
+          - name: I
+            template: whalesay
+            dependencies: [D, E, F]
+    - name: whalesay
+      container:
+        image: docker/whalesay:latest
```

Comment:
Not sure this should be in examples since it doesn't demonstrate any features. Is this a ui-test?

Reply:
yes, I can move it there